sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -466,9 +468,105 @@ def test_multiple_destinations_transform(self):
                max_file_size=20,
                max_files_per_bundle=-1))
 
+  @unittest.skip('TODO(BEAM-6611) - BQ file loads in streaming')
   @attr('IT')
-  def test_one_job_fails_all_jobs_fail(self):
+  def test_multiple_destinations_transform_streaming(self):
+    if isinstance(self.test_pipeline.runner, TestDataflowRunner):
+      self.skipTest("TestStream is not supported on TestDataflowRunner")
+    output_table_1 = '%s%s' % (self.output_table, 1)
+    output_table_2 = '%s%s' % (self.output_table, 2)
+    output_table_3 = '%s%s' % (self.output_table, 3)
+    output_table_4 = '%s%s' % (self.output_table, 4)
```

Comment:
Perhaps it makes sense to write only to two tables?

Reply:
Agreed. I'll write a bare bones test at first.